src/deployments/entities.ts

```diff
@@ -0,0 +1,30 @@
+import { Profile, Scene, Store, ValidateFunction, Wearable } from '@dcl/schemas'
+import { EntityType } from 'types'
+
+type AcceptedEntityType = Scene | Profile | Wearable | Store
+
+type Params = {
+  validate: ValidateFunction<AcceptedEntityType>
+  maxSize: number // in MB
```

Comment:
Why use MB here instead of bytes? It is not wrong but it is like using minutes for a timestamp, where you normally use milliseconds